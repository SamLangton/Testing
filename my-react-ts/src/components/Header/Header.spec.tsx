/*
	Snapshot tests for "./Header.tsx"
*/
import * as React from 'react';
import Header from './Header';
import * as renderer from 'react-test-renderer';

describe('./Header.tsx', () => {

	it('should render as expected', () => {
		const component = renderer.create(
			<Header raceInProgress={false} lastWinner={''}
			/>
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});