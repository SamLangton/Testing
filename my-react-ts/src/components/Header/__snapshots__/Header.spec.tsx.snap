// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./Header.tsx should render as expected 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
  />
  <h1
    className="App-title"
  >
    Implementation Race
  </h1>
  <div
    className="ScoreBoard"
  >
    <h2
      className="grads"
    >
      Kurtosys Grads
    </h2>
  </div>
</header>
`;
